src/pretix/presale/templates/pretixpresale/event/order_cancel.html

```diff
@@ -37,6 +50,7 @@ <h2>
             </p>
         {% endif %}
 
+        {% if not request.event.settings.cancel_allow_user_paid_require_approval or not request.event.settings.cancel_allow_user_paid_require_approval_fee_unknown %}
```

Comment:
This doesn’t allow the combination of requiring approval and allowing the customer to choose a lower refund amount. AFAICT this was possible before? I think the `cancel_allow_user_paid_require_approval` is in there, because the `cancel_allow_user_paid_require_approval_fee_unknown` should only be in effect, when `cancel_allow_user_paid_require_approval_fee_unknown` is actually set. So shouldn’t this be:

```suggestion
        {% if request.event.settings.cancel_allow_user_paid_require_approval and request.event.settings.cancel_allow_user_paid_require_approval_fee_unknown %}
        {% else %}
```

Reply:
I think your code and my code are equivalent, as De Morgan says:
```
(not A) or (not b) == not(a and b)
```

Reply:
You are absolutely right! Sorry for my confusion!